### DISCUSSION TITLE
Add configuration option to make slash trigger absolute path to the file system logic

### DISCUSSION THREAD

**Discussion by NerdFan** [mihai-vlc/path-autocomplete#119] [Ideas]
Hi!

I'm working with Linux target system via Remote SSH extension.
Path Autocomplete works with that setup but sometimes it requires to use absolute paths.
As "C:" is unusable in Linux environment then there is no way to use it at the moment.

Alternative idea: add option for relative path starting points like list. So current option can be by default listed but user can add own options.

**Comment by mihai-vlc** [comment]
Hi @NerdFan ,

Have you tried this setup ?

```json
    "path-autocomplete.pathMappings": {
        "/": "/"
    },
```
